apps/web/src/pages/Applications/ApplicationSkillsPage/components/SkillTree.tsx

```diff
@@ -31,13 +31,15 @@ interface SkillTreeProps {
   skill: Skill;
   experiences: Experience[];
   headingAs?: HeadingLevel;
+  hideConnectButton?: boolean;
```

Comment:
We may also want to add a `hideEdit?: boolean = false` or `editable?: boolean = true` prop to hide the edit link on the review page but continue to show it on the skill requirements page.

Reply:
Removed edit button on review page in [f580cc7](https://github.com/GCTC-NTGC/gc-digital-talent/pull/6600/commits/f580cc7fd5315c3affb5e39543538cacd21a1746)